ui/apps/platform/src/Containers/Integrations/IntegrationPage.tsx

```diff
@@ -22,14 +22,18 @@ import LinkShim from 'Components/PatternFly/LinkShim';
 import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
 import useIntegrationPermissions from './hooks/useIntegrationPermissions';
 import usePageState from './hooks/usePageState';
+import { Traits } from '../../types/traits.proto';
+import { TraitsOriginLabel } from '../AccessControl/TraitsOriginLabel';
+import { isUserResource } from '../AccessControl/traits';
```

Comment:
Same as previous comment and merge into other global imports in alphabetical order:
* Containers/AccessControl/TraitsOriginLabel
* Containers/AccessControl/traits
* types/traits.proto

Reply:
By the way, lines 16 through 19 above are a counter-example which might be written better as:

```ts
import {
    getEditDisabledMessage,
    getIntegrationLabel,
} from '../utils/integrationUtils';
```